src/app/types/institutional.endpoint.ts

```diff
@@ -1,5 +1,5 @@
 export interface Institutional {
   entityID: string
   Logos: any[]
-
+  DisplayNames: any[]
 }
```

Comment:
Loking into the DiscoFeed respone seems like the interface is incomplete
```
{
 "entityID": "http://iam.auckland.ac.nz/idp",
 "DisplayNames": [
  {
  "value": "The University of Auckland",
  "lang": "en"
  }
 ],
 "Descriptions": [
  {
  "value": "The University of Auckland Identity provider backed by the EPR system",
  "lang": "en"
  }
 ],
 "Logos": [
  {
  "value": "https://iam.auckland.ac.nz/logo/the-university-of-auckland-logo-tiny.png",
  "height": "132",
  "width": "320"
  }
 ]
},
```

Reply:
@dandineen This one should not be a blocker so not require to approve the PR. 
But generally is better to have a well-defined interface and use `any` only when the backend response is not predictable. 
So for the next interface, you have to define try to do a full endpoint description. This one is only used here so let's not worry too much about it.

Reply:
I already define the full endpoint.